src/QSymbolicsBase/basic_ops_homogeneous.jl

```diff
@@ -30,30 +30,39 @@ arguments(x::SScaled) = [x.coeff,x.obj]
 operation(x::SScaled) = *
 head(x::SScaled) = :*
 children(x::SScaled) = [:*,x.coeff,x.obj]
-Base.:(*)(c, x::Symbolic{T}) where {T<:QObj} = iszero(c) || iszero(x) ? SZero{T}() : SScaled{T}(c, x)
+function Base.:(*)(c, x::Symbolic{T}) where {T<:QObj} 
+    if iszero(c) || iszero(x)
+        SZero{T}()
+    else 
+        x isa SScaled ? SScaled{T}(c*x.coeff, x.obj) : SScaled{T}(c, x) 
+    end
+end
 Base.:(*)(x::Symbolic{T}, c) where {T<:QObj} = c*x
 Base.:(/)(x::Symbolic{T}, c) where {T<:QObj} = iszero(c) ? throw(DomainError(c,"cannot divide QSymbolics expressions by zero")) : (1/c)*x
 basis(x::SScaled) = basis(x.obj)
 
 const SScaledKet = SScaled{AbstractKet}
+maketerm(::Type{SScaledKet}, f, a, t, m) = f(a...)
```

Comment:
I think it is my bad on missing this new requirement of the API.

But is this using it correctly? In the docs I see it as 4-arg function, while here there are 5 args: https://juliasymbolics.github.io/TermInterface.jl/dev/

Reply:
also, if the vast majority are the same call can't we just write a single method along the lines of

```
maketerm(::Type{<:Symbolic{<:QObj}}, ...) = ...
```

Reply:
I don't think the docs for TermInterface are updated. The same can be said for most of the Symbolics libraries. It look some rummaging in the source code in SymbolicUtils to find out how Postwalk works. You can find it around the following line: https://github.com/JuliaSymbolics/SymbolicUtils.jl/blob/d009ca8e986b12982dce13794218ea6b632da617/src/rewriters.jl#L197.

Reply:
By the way, if you haven't noticed already, don't use this website for examining SymbolicUtils documentation: https://symbolicutils.juliasymbolics.org. Even though it comes up as on of the first results, it's wildly out of date.